website/source/api/auth/token/index.html.md

```diff
@@ -102,6 +102,10 @@ during this call.
 - `period` `(string: "")` - If specified, the token will be periodic; it will have
   no maximum TTL (unless an "explicit-max-ttl" is also set) but every renewal
   will use the given period. Requires a root/sudo token to use.
+- `entity_alias` `string: "")` - Name of the entity alias to associate with 
```

Comment:
This is missing an open paren. Also technically this should be listed as an array of strings, not a string, or both should be listed.

Reply:
Thanks. Fixed.